packages/dataviews/src/dataviews-layouts/table/index.tsx

```diff
@@ -407,10 +407,16 @@ function ViewTable< Item >( {
 					{ hasPrimaryColumn && (
 						<col className="dataviews-view-table__col-primary" />
 					) }
-					{ columns.map( ( column ) => (
+					{ columns.map( ( column, index ) => (
 						<col
 							key={ `col-${ column }` }
-							className={ `dataviews-view-table__col-${ column }` }
+							className={ clsx(
+								`dataviews-view-table__col-${ column }`,
+								{
+									'dataviews-view-table__col-primary':
```

Comment:
I think the fix is fine. Should we rename the `dataviews-view-table__col-primary` classname though to indicate that is the first 'data' column (as we exclude the checkbox). `dataviews-view-table__col-primary` is only used for applying the width so it might be cleaner to rename appropriately to match the semantics.

Reply:
I think `dataviews-view-table__col-expand` would be a good fit for the classname.

Reply:
I agree, decoupling the name from “primary” makes sense. Something like `__col-first-data` communicates the behavior without implying semantic meaning. Any of the variants you proposed would work for me.

Reply:
There was also a previous disconnect between the classname and the API: in the API, we set `titleField` while in the classname, we use `primary` nomenclature. Noting that in actions, we use `isPrimary`.